index.html

```diff
@@ -4,9 +4,90 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
     <title>Project: HTML Forms</title>
 </head>
 <body>
-    
+    <div class="sign-in">
+        <div class="sign-text">
+            <p>Already have an Intuit account?</p>
+            <a href="">I forgot my User ID or Password</a>
+        </div>
+        <button><img src="images/lock.png"> <p>Sign in</p></button>
+    </div>
+    <div class="title-form">
+        <div class="title">
+            <img src="images/intuit.png" alt="">
+        </div>
+        <div class="apps">
+            <div class="diferent-apps">
+                <img class="mint" src="images/mint.png" alt="">
+                    <p>Mint</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="quickbooks" src="images/quickbooks.png" alt="">
+                    <p>quickbooks</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="turbo" src="images/turbo.png" alt="">
+                    <p>TurboTax</p>
+            </div>    
+        </div>
+    </div>
+    <div class="form">
+        <form action="">
```

Comment:
The action attribute for form should not be left empty. Remove the action if there's no controller to handle the form.